exporters/stdout/stdoutmetric/exporter.go

```diff
@@ -51,13 +51,6 @@ func (e *exporter) Aggregation(k metric.InstrumentKind) metric.Aggregation {
 }
 
 func (e *exporter) Export(ctx context.Context, data *metricdata.ResourceMetrics) error {
-	select {
-	case <-ctx.Done():
-		// Don't do anything if the context has already timed out.
-		return ctx.Err()
-	default:
-		// Context is still valid, continue.
-	}
```

Comment:
I think this might be useful as there is later no code (when exporting) that handles cancellation.

This could be refactored to

```go
	if err := ctx.Err(); err != nil {
		return err
	}
```

Same comment applies to `stdouttrace.Exporter.ExportSpans`  - it may be good to add such check.

Reply:
Makes sense - PR updated. Please review.